packages/eas-cli/src/commands/branch/publish.ts

```diff
@@ -1,51 +1,100 @@
 import { getConfig } from '@expo/config';
 import { Command, flags } from '@oclif/command';
+import { uniqBy } from '@oclif/plugin-help/lib/util';
```

Comment:
Not sure why this is imported from `@oclif/plugin-help`, maybe this was added automatically?

Reply:
Yea good point.